src/modules/assessments/components/AssessmentFormSideBar.tsx

```diff
@@ -136,11 +130,6 @@ const AssessmentFormSideBar: React.FC<Props> = ({
                 onSuccess={navigateToEnrollment}
               />
             )}
-            {showAssessmentId && (
-              <Typography variant='body2'>
-                <b>Assessment ID:</b> {assessment.id}
```

Comment:
Remove this, it is only shown in development (`import.meta.env.MODE === 'development'`) and not useful, since the id is in the URL bar